rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -19,9 +19,79 @@ add a new schema rather than change existing material.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
--->
+The proposed change adds six fields, as described below:
+
+```
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster
+      level: extended
+      type: object
+      description: >
+        Orchestrator cluster details.
+
+    - name: cluster.name
+      level: extended
+      type: keyword
+      description: >
+        Name of the cluster.
+
+    - name: cluster.url
+      level: extended
+      type: keyword
+      description: >
+        URL of the cluster.
+
+    - name: type
+      level: extended
+      type: keyword
+      example: kubernetes
+      description: >
+        Orchestrator cluster type (e.g. kubernetes, nomad or cloudfoundry).
+
+    - name: organization
+      level: extended
+      type: keyword
+      example: elastic
+      description: >
+        Organization affected by the event (for multi-tenant orchestrator setups).
+
+    - name: namespace
+      level: extended
+      type: keyword
+      example: kube-system
+      description: >
+        Namespace in which the action is taking place.
+
+    - name: resource.name
+      level: extended
+      type: keyword
+      example: test-pod-cdcws
+      description: >
+        Name of the resource being acted upon.
+
+    - name: resource.type
+      level: extended
+      type: keyword
+      example: service
+      description: >
+        Type of resource being acted upon.
+
+    - name: api_version
```

Comment:
how about adding `orchestrator.version` too? I see value to know what version of Kubernetes cluster I run on (ie 1.18 vs 1.19 etc)

Reply:
Good point @ChrsMark! Or maybe this field can also go under `orchestrator.cluster` to be `orchestrator.cluster.version`?

Reply:
Ahh maybe we should keep `orchestrator.version` at the same level as `orchestrator.type`. I don't have a strong opinion here :)

Reply:
The `version` field is a good shout - I'll add it under the `cluster` object, as if we're defining that to contain all the cluster-wide fields, `version` definitely fits in that category.